enterprise/permanentpen.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Permanent Pen Plugin
+title_nav: Permanent Pen
+description: Apply formats while typing
+keywords: permanent pen copy text format style
+controls: toolbar button, contextmenu, menu item
+---
+
+Permanent Pen is a premium TinyMCE plugin for applying a pre-defined format (color, typeface and type style, or font) to a desired section of content. The ability to store information for reuse saves the user time and ensures consistency across the content.
+
+The Permanent Pen allows the user to add comments without modifying the original content.
```

Comment:
Is "comments" the best word here? I still prefer the term "mark up" since it's more general, though I'm not sure that's correct either.  Maybe something like "The Permanent Pen allows the user to add formatted mark up without modifying the original content, which is useful for reviewing and commenting on content."? Although then we're getting into use cases again...